src/parser/command.cpp

```diff
@@ -99,6 +99,19 @@ define_transition_command::define_transition_command(std::string id, system::tra
 {
 }
 
+void define_process_type_command::run(system::context* ctx, engine* e) {
+	/* FIXME something should go there */
```

Comment:
Yes, something should go here. I guess process type should be added to the context, or something like that.